components/footer/ChatWindow.js

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) 2006-present TrueChoice IP Holding Company, Inc.
+ * All rights reserved.
+ *
+ * This software is the confidential and proprietary information of TrueChoice IP Holding Company, Inc.
+ * ("Confidential Information").  You shall not disclose such Confidential Information and shall
+ * use it only in accordance with the terms of the license agreement you entered into with the company.
+ */
+
+// dependencies
+import { useState } from 'react';
+import { Card, Form, FormLabel, FormControl } from 'react-bootstrap';
+import { X } from 'react-bootstrap-icons';
+// local files
+
+/**
+ * @description - Chat Window display w/ user & bot interactions
+ * @param {string} props.cardClassName - class names for Card bootstrap component
+ * @param {function} props.toggleChatWindowDisplay - toggle "open" state. Functionality: on X icon click, close window
+ * @return {jsx} - the ChatWindow component to render
+ */
+const ChatWindow = ({ cardClassName = 'bg-dark', toggleChatWindowDisplay }) => {
+  // state
+  const [userText, setUserText] = useState('');
+  const [userMessages, setUserMessages] = useState([]);
+
+  if (userMessages.length > 3) {
+    setUserMessages([...userMessages.slice(1)]);
+  }
+
+  // event handlers
+  /**
+   * @description - on change event listener that sets "userText" state to user's input value
+   * @param {object} event - event object, accessing the event.target.value to get user's text
+   * @return {undefined} - Nothing to return
+   */
+  const handleInputChange = (event) => {
+    setUserText(event.target.value);
+  };
+
+  /**
+   * @description - form submit event listener. 1) prevents default refreshing of page when submitting form
+   * 2) After submit, sets the text input to blank. 3) Sets "userMessages" state to include
+   * previous & current submitted texts
+   * @param {object} event - event object, accessing preventDefault method to prevent default page reload
+   * @return {undefined} - Nothing to return
+   */
+  const handleFormSubmit = (event) => {
+    event.preventDefault();
+    setUserText('');
+    setUserMessages([...userMessages, userText]);
+  };
+
+  // render content
+  const renderUserMessages = userMessages.map((userMessage, i) => (
```

Comment:
is this a function or a variable? since the name is `renderUserMessages` (starts with a verb), it seems like it is a function - in which case, we need a function description

Reply:
@achange-tcs Ready for review! - Same as HelpChat.js